src/helpers.ts

```diff
@@ -1,19 +1,23 @@
+import type { MarkedOptions, Rule } from './MarkedOptions.ts';
+import type { ResultCallback } from './marked.js';
+
 /**
  * Helpers
  */
 const escapeTest = /[&<>"']/;
 const escapeReplace = new RegExp(escapeTest.source, 'g');
 const escapeTestNoEncode = /[<>"']|&(?!(#\d{1,7}|#[Xx][a-fA-F0-9]{1,6}|\w+);)/;
 const escapeReplaceNoEncode = new RegExp(escapeTestNoEncode.source, 'g');
-const escapeReplacements = {
+const escapeReplacements: Record<string, string> = {
```

Comment:
```suggestion
const escapeReplacements = {
```

Not sure why this is needed

Reply:
You are right. This also works without explicit type. Removed it.